api/query/params.go

```diff
@@ -39,14 +37,14 @@ func (p *Params) Decode(u *url.URL) error {
 		switch key {
 		case "tag":
 			for _, value := range values {
-				tag := strings.Split(value, ":")
-				if len(tag) != 2 {
+				if i := strings.Index(value, ":"); i != -1 {
+					p.tags[value[:i]] = append(p.tags[value[:i]], value[i+1:])
+				} else {
 					return fmt.Errorf("invalid query tag value: %s", value)
 				}
-				p.tags[tag[0]] = append(p.tags[tag[0]], tag[1])
 			}
-		case amicache.StateTag:
-			p.tags[amicache.StateTag] = append(p.tags[amicache.StateTag], values...)
+		case stateTag, "state", "status": // aliases for the state tag
+			p.tags[stateTag] = append(p.tags[stateTag], values...)
```

Comment:
@jmassara is status just so we are backwards compatible?

Reply:
Yes, it was previously hard-coded to `status`.